pkg/lib/interaction/nodes/remove_authenticator.go

```diff
@@ -35,6 +36,14 @@ func (e *EdgeRemoveAuthenticator) Instantiate(ctx *interaction.Context, graph *i
 		return nil, err
 	}
 
+	if info.UserID != graph.MustGetUserID() {
```

Comment:
Prefer to check in authenticator.Service

Reply:
Authenticator facade is used by other transport e.g. userimport. In those cases deleting from other/no user id should be legitimate?

Reply:
I see.
BTW, is there same problem in delete identity?

Reply:
Found have similar issue in remove and update identity as well. Let me update that thanks!